docs/content/en/docs/clustermgmt/cluster-status.md

```diff
@@ -77,7 +77,7 @@ Conditions provide a high-level status report representing an assessment of clus
   * `DefaultCNIConfigured` - reports the configuration state of the default CNI specified in the cluster specifications. It will be marked as `True` once the default CNI has been successfully configured on the cluster. 
   However, if the EKS Anywhere default cilium CNI has been [configured to skip upgrades]({{< relref "../getting-started/optional/cni/#use-a-custom-cni" >}}) in the cluster specification, then this condition will be marked as `False` with the reason `SkipUpgradesForDefaultCNIConfigured`.
 
-  * `WorkersReady` - reports that the condition of the current state of worker machines versus the desired state specified in the Cluster specification. This condition is marked `True` once the number of worker nodes in the cluster match the expected number of worker nodes as defined in the cluster specifications and all the worker nodes are up to date and ready.
+  * `WorkersReady` - reports that the condition of the current state of worker machines versus the desired state specified in the Cluster specification. This condition is marked `True` once the number of worker nodes in the cluster match the expected number of worker nodes as defined in the cluster specifications and all the worker nodes are up to date and ready for worker node groups not having [autoscaling]({{< relref "../getting-started/optional/autoscaling" >}}) configuration as well as the number of worker nodes in the cluster lies between the minCount and maxCount number of replicas defined in the cluster specification for worker node groups having autoscaling configuration.
```

Comment:
We should say something like "once the following conditions are met:
1. For worker node groups with autoscaling configured, ...
2. For fixed worker node groups, ..."